### PR TITLE
:bug: fix multiple netsparker issues

### PR DESCRIPTION
**Pull request by manuel-sommer**
- https://github.com/DefectDojo/django-DefectDojo/issues/9816
- cvss31 was missing 
- item["State"] can be None

### CODE REVIEW

File: dojo/tools/netsparker/parser.py [hunk 1]
```diff
@@ -60,21 +65,20 @@ def get_findings(self, filename, test):
...
-            if item["State"].find("AcceptedRisk") != -1:
+            elif state == "AcceptedRisk":
                 finding.risk_accepted = True
 
             if (
                 (item["Classification"] is not None)
-                and (item["Classification"]["Cvss"] is not None)
-                and (item["Classification"]["Cvss"]["Vector"] is not None)
+                and (item["Classification"].get("Cvss") is not None)
+                and (item["Classification"].get("Cvss").get("Vector") is not None)
```
Comment (kiblik) [inline dojo/tools/netsparker/parser.py:81]: There is possibly a bit more "compressed version"
```python
if not item.get("Classification", {}).get("Cvss", {}).get("Vector"):
```

Comment (manuel-sommer) [reply dojo/tools/netsparker/parser.py:81]: https://github.com/DefectDojo/django-DefectDojo/blob/c182e9ca9d8f981c15de2018f948fe69c4d1a800/unittests/scans/netsparker/netsparker_many_findings.json#L4224 --> AttributeError: 'NoneType' object has no attribute 'get'

Comment (kiblik) [reply dojo/tools/netsparker/parser.py]: Ou. I'm sorry, you are right

Comment (manuel-sommer) [reply dojo/tools/netsparker/parser.py]: No problem, I understand that you did not go through all unittest files. Good idea from your side.

**Review by Maffooch — APPROVED** [review]

**Review by mtesauro — APPROVED** [review]
Approved

**Review by grendel513 — APPROVED** [review]

**Review by blakeaowens — APPROVED** [review]

**Review by cneill — APPROVED** [review]

**Pull request merged**